docs/modules/ROOT/pages/reference/hierarchy.adoc

```diff
@@ -0,0 +1,89 @@
+= Configure the inventory hierarchy
+
+The https://kapitan.dev/inventory/[Kapitan inventory] used by Commodore builds on top of https://reclass.pantsfullofunix.net/[reclass].
+Reclass allows to do interpolation of parameters.
```

Comment:
Note: Upstream Reclass doesn't allow parameters in classes. That's one of the features of Kapitan's fork: https://github.com/kapicorp/reclass/blob/develop/README-extensions.rst#use-references-in-class-names

Reply:
Note: It's not really the Kapicorp fork, but the salt-formula fork: https://github.com/salt-formulas/reclass
(which in turn is a fork of the original madduck reclass: https://github.com/madduck/reclass)

Reply:
I have now explicitly stated, that a fork is being used. Including link to the extension documentation of that fork.